jscomp/bsb/bsb_merlin_gen.ml

```diff
@@ -149,21 +143,45 @@ let merlin_file_gen ~(per_proj_dir : string)
         Buffer.add_string buffer (merlin_flg_pp ^ x));
     Buffer.add_string buffer
       (merlin_flg_ppx
+      ^ (match (reason_react_jsx, jsx_version) with
```

Comment:
Here it's better to keep on using `Buffer.add_string` instead of concatenating all these string.

Reply:
Actually, generating a `.merlin` file is a leftover thing that's not needed anymore.
No need for this, I'll remove the file in a separate PR.

Reply:
Here: https://github.com/rescript-lang/rescript-compiler/pull/5504

Reply:
Got it I'll rebase to master.